Add input device stubs (#3411)
ppy/osu-wiki: wiki/Gameplay/Input_device/en.md

# Input device

> **Input device** refers to the device used to play osu!. 
> This can be a mouse, keyboard or tablet. osu! supports many input devices.

Reviewer: ```suggestion
An **input device** is a device that is used to contrl input on a computer or specific piece of software. In the osu! wiki, it is used to *specifically refer* to inputs that are used in osu! The follow is a list of some of the many common input devices used for osu!:
```

> - [Mouse](/wiki/Gameplay/Input_device/Mouse)
> - [Keyboard](/wiki/Gameplay/Input_device/Keyboard)
> - [Touch Device](/wiki/Gameplay/Input_device/Touch_device)
> - [Graphic Tablet](/wiki/Gameplay/Input_device/Graphic_tablet)
> - [Wiimote](/wiki/Gameplay/Input_device/Wiimote)
> - [Drum Controller](/wiki/Gameplay/Input_device/Drum_controller)
> - [Dance Pad](/wiki/Gameplay/Input_device/Dance_pad)
> - [Bemani Controller](/wiki/Gameplay/Input_device/Bemani_controller)

Reviewer: ```suggestion
- [Mouse](/wiki/Gameplay/Input_device/Mouse)
- [Keyboard](/wiki/Gameplay/Input_device/Keyboard)
- [Touch device (touchscreen)](/wiki/Gameplay/Input_device/Touch_device)
- [Graphics tablet](/wiki/Gameplay/Input_device/Graphic_tablet)
- [Wiimote](/wiki/Gameplay/Input_device/Wiimote)
- [Drum controller/Taiko controller](/wiki/Gameplay/Input_device/Drum_controller)
- [Dance pad/dance mat](/wiki/Gameplay/Input_device/Dance_pad)
- [Bemani controller](/wiki/Gameplay/Input_device/Bemani_controller)
```

- [Wiimote](/wiki/Gameplay/Input_device/Wiimote)
- [Drum Controller](/wiki/Gameplay/Input_device/Drum_controller)
- [Dance Pad](/wiki/Gameplay/Input_device/Dance_pad)

> - [Beatmania Controller](/wiki/Gameplay/Input_device/Beatmania_controller)

Reviewer: ```suggestion
- [Beatmania controller](/wiki/Gameplay/Input_device/Beatmania_controller)
```
